src/aggregation/bucket/histogram/histogram.rs

```diff
@@ -117,6 +117,9 @@ pub struct HistogramAggregation {
     /// Cannot be set in conjunction with min_doc_count > 0, since the empty buckets from extended
     /// bounds would not be returned.
     pub extended_bounds: Option<HistogramBounds>,
+    /// Whether to return the buckets as a hash map
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub keyed: Option<bool>,
```

Comment:
why `Option<bool>` instead of bool

Reply:
Used Option following the other optional params and to make it easier to omit by `Default::default()` but no strong reason.
Should we use `bool`?

Reply:
`Default::default()` should also work on bool (defaults to false). Since `None` equals `false`, I think it's better to use bool directly

Reply:
Oh I see, that makes sense!  Let me change those to `bool`!

Reply:
fixed in https://github.com/quickwit-oss/tantivy/pull/1424/commits/5ab5f070edb5ae7e2f00daeb626e517e8290d83a